tiny_bvh.h

```diff
@@ -581,6 +636,17 @@ class BVH
 
 namespace tinybvh {
 
+void * BVH::Alloc(size_t size)
```

Comment:
The function name does not point out the cache line alignment functionality. Maybe call it 'AlignedAlloc / AlignedFree'?